src/versioning/Versioning.ts

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import DefaultBrowserBehavior from '../browserbehavior/DefaultBrowserBehavior';
+import { hash, semverString } from './version.json';
```

Comment:
Do we need to check in version.json?

Reply:
No. It's specifically ignored in `.gitignore` so that nobody accidentally checks it in. It's just how we get the contents of `git describe` into the build output.